index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <span><a href="#">My Logo</a></span>
+            <span><a href="#"><img src="images/Union.png" alt="mobile navigation"></a></span>
+        </nav>
```

Comment:
- [ ] as mentioned by the previous reviewer `Also make sure to wrap the hamburger icon on the nav tag, not all the content of the section.` what he meant is you should only put the mobile menu icon inside the nav tag and move the Logo outside of it, kindly fix it 🙏  if you need assistance contact me on slack @Ali Jendoubi